drivers/interrupt_controller/Kconfig.it8xxx2

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2015 ITE Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config ITE_IT8XXX2_INTC
+	bool "ITE_IT8XXX2 Interrupt controller"
+	depends on (SOC_IT8XXX2)
```

Comment:
Should we set a default value here that would work in most cases?

Reply:
Yes, this value is default selected by soc\riscv\riscv-ite\it8xxx2\Kconfig.defconfig.series.